packages/babel-helpers/src/helpers/toPropertyKey.js

```diff
@@ -0,0 +1,9 @@
+/* @minVersion 7.1.5 */
+
+// https://tc39.es/ecma262/#sec-topropertykey
+import toPrimitive from "toPrimitive";
+
+export default function _toPropertyKey(arg) {
+  var key = toPrimitive(arg, "string");
+  return typeof key === "symbol" ? key : String(key);
```

Comment:
Further assumption idea: `_toPropertyKey` can be simplified to `typeof key === "symbol" ? key : "".concat(key)` if `[Symbol.toPrimitive]` method does not return a symbol.